weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterBar.tsx

```diff
@@ -209,26 +207,26 @@ export const FilterBar = ({
 
   return (
     <>
-      <div
+      <Button
```

Comment:
The teal background on hover doesn't feel that great here, esp. when you have a ref link as the filter value. Let's discuss with design.